src/tabs/index.tsx

```diff
@@ -47,6 +47,9 @@ export default function Tabs({
   }
   const { __internalRootRef } = useBaseComponent('Tabs', {
     props: { disableContentPaddings, variant },
+    metadata: {
+      hasActions: tabs.some(tab => !!tab.action),
```

Comment:
Let's also add fitHeight as props and disabled reason as metadata so that the component is better covered